akka-actor/src/main/resources/reference.conf

```diff
@@ -1061,6 +1061,17 @@ akka {
         # How often to sweep out expired cache entries.
         # Note that this interval has nothing to do with TTLs
         cache-cleanup-interval = 120s
+
+        # Configures the list of search domains.
+        # Defaults to a system dependent lookup (on Unix like OSes, will attempt to parse /etc/resolv.conf, on
+        # other platforms, will not make any attempt to lookup the search domains). Set to a single domain, or
+        # a list of domains, eg, [ "example.com", "example.net" ].
```

Comment:
👍 I like the example values for expected syntax (e.g. no trailing `.`: `example.net.`)